gadget/layout.go

```diff
@@ -243,29 +243,50 @@ func LayoutVolume(gadgetRootDir string, volume *Volume, constraints LayoutConstr
 	return vol, nil
 }
 
+type multiErr []error
+
+func (me multiErr) String() string {
+	s := ""
+	for i, e := range me {
+		s += "- " + e.Error()
+		if i < len(me)-1 {
+			s += "\n"
+		}
+	}
+	return s
+}
+
 // ResolveContentPaths resolves any "$kernel:" refs in the gadget
 // content and populates VolumeContent.resolvedSource with absolute
 // paths.
 //
 // XXX: maybe move into LayoutVolume(), operator on *Volume and make private?
-func ResolveContentPaths(lv *LaidOutVolume, gadgetRootDir, kernelRootDir string) error {
+func ResolveContentPaths(lv *Volume, gadgetRootDir, kernelRootDir string) error {
```

Comment:
why this change?

Reply:
I also wonder if either Volume or LaidOutVolume should carry a flag to know whether it was resolved or not, we could use that early in functions that must operate only on resolved volumes